exercises/07-Buttons-Alert-And-Table/README.es.md

```diff
@@ -1,28 +1,29 @@
-# `07` Botones, alerta, iconos y tablas
+# `07` Buttons, Alert, Glyphicon and Table
```

Comment:
Remove Glyphicon as you did in the english version

Reply:
Done